shell/browser/api/electron_api_view.cc

```diff
@@ -261,7 +256,10 @@ void View::RemoveChildView(gin::Handle<View> child) {
   if (!view_)
     return;
 
-  const auto it = base::ranges::find(child_views_, child.ToV8());
+  const auto it =
+      std::ranges::find_if(child_views_, [&](const ChildPair& child_view) {
+        return child_view.second == child.ToV8();
+      });
```

Comment:
I'd like you to check my work; but I _think_ this test would work and is slightly cheaper than the `ToV8()` call

```suggestion
  const auto it =
      std::ranges::find_if(child_views_, [&](const ChildPair& child_view) {
        return child_view.first == child->view();
      });
```

Reply:
@ckerr I don't think that will work because in this case child is gin::Handle<View>.

Reply:
Yup that works!